### ISSUE TITLE
Add accessibility section

### ISSUE THREAD

**Issue by LJWatson** [w3c/IntersectionObserver#420]
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15
